src/Language/PureScript/TypeChecker/Roles.hs

```diff
@@ -206,13 +206,15 @@ inferDataDeclarationRoles moduleName (tyName, tyArgs, ctors) roleEnv =
           -- arguments, recursively infer the roles of the type constructor's
           -- arguments. For each (role, argument) pair:
           --
-          -- * If the role is nominal, mark all free variables in the
-          --   argument as nominal also, since they cannot be coerced if the
-          --   argument's nominality is to be preserved.
-          -- * If the role is representational, recurse on the argument, since
-          --   its use of our parameters is important.
-          -- * If the role is phantom, terminate, since the argument's use of
-          --   our parameters is unimportant.
+          -- If the role is nominal,
```

Comment:
Can you tidy up the line wrapping here please?

Reply:
I delete too much newline.